chart/compass/values.yaml

```diff
@@ -72,7 +72,7 @@ global:
       version: "PR-1939"
     pairing_adapter:
       dir:
-      version: "PR-1939"
+      version: "PR-1955"
```

Comment:
Changes in the charts don't require version update. The versions should be updated only when there are changes in the code of a component - a new image is built in that case, and we refer to that new image here.